nssp_demo/forecast_state.py

```diff
@@ -49,26 +49,41 @@ def main(
     disease: str,
     report_date: str,
     state: str,
-    nssp_data_dir: Path | str,
+    facility_level_nssp_data_dir: Path | str,
+    state_level_nssp_data_dir: Path | str,
     param_data_dir: Path | str,
     output_data_dir: Path | str,
     n_training_days: int,
     n_forecast_days: int,
     n_chains: int,
     n_warmup: int,
     n_samples: int,
-    last_training_date: str,
+    last_training_date: str = None,
```

Comment:
I think I would prefer to parameterize this in terms of days prior to `report_date`

Reply:
I went back and forth on this. Happy to switch to measuring it relative to the report date.

Reply:
Changed in 9e87af9